priv/protos/eigr/functions/protocol/actors/protocol.proto

```diff
@@ -226,9 +226,11 @@ message RegistrationResponse {
 //
 // Params:
 //   * state: Actor state passed back and forth between proxy and user function. 
+//   * name: Actor name, having this data here is very useful for abstract actors
 message Context {
 
     google.protobuf.Any state = 1;
+    string name = 2;
```

Comment:
it would be interesting to add the actor that is invoking and not just the invoked actor.
I think it would be interesting to use:

```proto
message Context {
  google.protobuf.Any state = 1;

  ActorId from = 2;

  ActorId self = 3;
}